rts/lra/pom.xml

```diff
@@ -25,12 +25,14 @@
         <version.cdi-api>1.2</version.cdi-api>
         <version.junit>4.12</version.junit>
 
-        <version.thorntail>2.4.0.Final</version.thorntail>
         <version.arquillian>1.2.1.Final</version.arquillian> <!-- cannot use the up-to-date Arquillian https://issues.jboss.org/browse/THORN-2090 -->
 
+        <version.microprofile.lra.api>1.0-20190614.061200-417</version.microprofile.lra.api>
+        <version.microprofile.lra.tck>1.0-20190614.061201-417</version.microprofile.lra.tck>
+<!--
         <version.microprofile.lra.api>1.0-20190515.061205-378</version.microprofile.lra.api>
         <version.microprofile.lra.tck>1.0-20190515.061206-378</version.microprofile.lra.tck>
-
+-->
```

Comment:
this comment can be removed also I think

Reply:
I'm not ready yet - I will clean up the PR when I've seen the TCK passing